### PR TITLE
[RSDK-5198] Change units in UR driver to match config field units

### PR DESCRIPTION
**Pull request by randhid**
What it says on the tin.

Max speed for ur5e from spec: https://www.universal-robots.com/media/1807465/ur5e-rgb-fact-sheet-landscape-a4.pdf
Movej API in URSCRIPT takes in rad/s for velocity and rad/s^2 for acceleration: https://s3-eu-west-1.amazonaws.com/ur-support-site/115824/scriptManual_SW5.11.pdf

- [x] Creating an app image too - I'd like someone to test this out on a ur5 in lab while I'm remote.

### CODE REVIEW

File: components/arm/universalrobots/ur.go [hunk 1]
```diff
@@ -46,7 +48,7 @@ func (cfg *Config) Validate(path string) ([]string, error) {
 	if cfg.Host == "" {
 		return nil, goutils.NewConfigValidationFieldRequiredError(path, "host")
 	}
-	if cfg.Speed > 1 || cfg.Speed < .1 {
+	if cfg.Speed > 180 || cfg.Speed < 18 {
 		return nil, errors.New("speed for universalrobots has to be between .1 and 1")
```
Comment (randhid) [inline components/arm/universalrobots/ur.go:52]:
```suggestion
		return nil, errors.New("speed for universalrobots has to be between 18 and 180 degrees per second")
```

File: components/arm/universalrobots/ur.go [hunk 2]
```diff
@@ -165,10 +167,6 @@ func URArmConnect(ctx context.Context, conf resource.Config, logger golog.Logger
 		return nil, err
 	}
 
-	if newConf.Speed > 1 || newConf.Speed < .1 {
```
Comment (randhid) [inline components/arm/universalrobots/ur.go:168]: Removed repeated validation check.

**Review by JessamyT — APPROVED** [review]
LGTM but I just write things :) Thx for quick fix. I'm already making the docs change so don't worry about that ticket.

**Review by oliviamiller — APPROVED** [review]
lgtm lmk if you want me to test on the UR!

**Comment by randhid** [comment]
> lgtm lmk if you want me to test on the UR!

Yes please if you have time!

File: components/arm/universalrobots/ur.go (hunk 1 above)
Comment (raybjork) [reply components/arm/universalrobots/ur.go:52]: Yeah we'll want to update this

File: components/arm/universalrobots/ur.go [hunk 3]
```diff
@@ -46,7 +48,7 @@ func (cfg *Config) Validate(path string) ([]string, error) {
 	if cfg.Host == "" {
 		return nil, goutils.NewConfigValidationFieldRequiredError(path, "host")
 	}
-	if cfg.Speed > 1 || cfg.Speed < .1 {
+	if cfg.Speed > 180 || cfg.Speed < 18 {
```
Comment (raybjork) [inline components/arm/universalrobots/ur.go:51]: Can we rename the config parameter to be `SpeedDegsPerSec

File: components/arm/universalrobots/ur.go [hunk 4]
```diff
@@ -422,8 +420,8 @@ func (ua *URArm) MoveToJointPositionRadians(ctx context.Context, radians []float
 		radians[3],
 		radians[4],
 		radians[5],
-		5.0*ua.speed,
-		4.0*ua.speed,
```
Comment (raybjork) [inline components/arm/universalrobots/ur.go:426]: Its concerning that we just weren't doing this to begin with

**Review by raybjork — CHANGES_REQUESTED** [review]
Thanks! Just looking for one change here

**Review by raybjork — APPROVED** [review]

**Pull request merged**